README.md

```diff
@@ -1,24 +1,48 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+# Kitty News API
+
+## Description
+This is the API to our project Kitty News. Here we are authenticating users, storing articles, saving all our data and also making calls to external APIs such as Stripe. This is built with Ruby on Rails. We are using Amazon Web Services to store our images.
+- User registration
+- User authentication(Log-in/Log-off)
+- Yearly subscription plan
+- Display articles in a category
+
+## The code
+Deployed App: (https://kitty-news.herokuapp.com/) </br>
+Github: (https://github.com/CraftAcademy/kitty_news_api)</br>
+The admin user interface, mobile application and User Interface can be found here:
+  - [User Interface](https://github.com/CraftAcademy/kitty_news_client_user)
+  - [Admin User Interface](https://github.com/CraftAcademy/kitty_news_client_admin)
+  - [Mobile](https://github.com/CraftAcademy/kitty_news_client_mobile)
+
+## User stories
+Check the whole design sprint and user story history of the project in [this](https://www.pivotaltracker.com/n/projects/2479403) Pivotal Tracker board.
+
+## Dependencies
+- Ruby
+- Rails
+- RSpec
+- Puma
+- Pry
+- Coveralls
+- Factory Bot
+- Shouda-Matchers
+- Rack-cors
+- Active_model_serializers
+- Devise_token_auth
+- Stripe-rails
+- Aws-sdk-s3
+
+## Acknowledgements
+- [Craft Academy](https://www.craftacademy.se/) course material
+- [Craft Academy](https://www.craftacademy.se/) coaches: Thomas Ochman, Emma-Maria Thalen
+
+## License
+MIT License
+
+## Authors
+- [Sanne Lindholm](https://github.com/salindholm)
+- [Kyungin Na](https://github.com/KyunginNa)
+- [Viktor Karlsson](https://github.com/ViktorHek)
+- [Erik](https://github.com/Ehugo2000)
```

Comment:
Found out what Erik's full name is! [Erik Hugoson]

Reply:
Will add that!